source/Irrlicht/CIrrDeviceSDL.cpp

```diff
@@ -218,9 +218,20 @@ int CIrrDeviceSDL::findCharToPassToIrrlicht(int assumedChar, EKEY_CODE key) {
 void CIrrDeviceSDL::resetReceiveTextInputEvents() {
 	gui::IGUIElement *elem = GUIEnvironment->getFocus();
 	if (elem && elem->acceptsIME())
+	{
 		SDL_StartTextInput();
+		core::rect<s32> pos = elem->getAbsolutePosition();
+		SDL_Rect rect;
+		rect.x = pos.UpperLeftCorner.X;
+		rect.y = pos.UpperLeftCorner.Y;
+		rect.w = pos.LowerRightCorner.X - rect.x;
```

Comment:
Use [`pos.getWidth()`](https://irrlicht.sourceforge.io/docu/classirr_1_1core_1_1rect.html#ae313a3674095733348d5e8bcb75c49b4) here

Reply:
Thanks for pointing this out. I mostly took the code snippet from its `CIrrDeviceLinux` counterpart and mostly expected this to be a quick fix.